plugin/pkg/global/resourcereferencemanager/admission.go

```diff
@@ -481,3 +546,12 @@ func (r *ReferenceManager) lookupSecret(namespace, name string) error {
 
 	return nil
 }
+
+func getRemovedVersions(oldVersions []core.ExpirableVersion, newVersions []core.ExpirableVersion) sets.String {
```

Comment:
Why don't you reuse your helper function? https://github.com/gardener/gardener/pull/2105/files#diff-0a71cc18fe106e50d1a5aca2b5330aafR173

Reply:
Thats whats being done one line below? 
[As mentioned here](https://github.com/gardener/gardener/pull/2105#discussion_r399186975) I will use the returned set.Set from the updated helper to get rid of the then redundant code below.